### PR TITLE
Added `npm config set save true` tip

### PR DESCRIPTION
**Pull request by twolfson**
One tip I have found useful is to always save packages by default with `save=true` in `.npmrc`. In this PR:
- Added tip about saving packages by default (e.g. without `--save` or `--save-dev`)

### CODE REVIEW

**Comment by isaacs** [comment]
The wording is a bit rough here with the double-negative ("prevent forgetting..."). Can you rephrase it so that it's a positive statement that is more easily parsed?

**Comment by twolfson** [comment]
Sure thing, how about:

```
Always save to `package.json` on `npm install` with `npm config set save true`
```
